platform/iphone/export/export.cpp

```diff
@@ -1290,6 +1292,18 @@ Error EditorExportPlatformIOS::_export_ios_plugins(const Ref<EditorExportPreset>
 			p_config_data.capabilities.push_back(capability);
 		}
 
+		// Linker flags
+		// Checking duplicates
+		for (int j = 0; j < plugin.linker_flags.size(); j++) {
+			String linker_flag = plugin.linker_flags[j];
+
+			if (plugin_linker_flags.find(linker_flag) != -1) {
+				continue;
+			}
+
+			plugin_linker_flags.push_back(linker_flag);
+		}
```

Comment:
If you use a HashSet here you won't need to check for duplicates.

Reply:
I'm not sure this type of container exists in Godot's source - `error: unknown type name 'HashSet'`

Reply:
Turns out I was misremembering (too used to using HashSet at work I guess), I know there is a Set and a TreeSet, pretty sure they also implement unique entries there but may be wrong.
If those don't work consider my comment void.